src/integrations/FacebookPixel/browser.js

```
@@ -80,42 +104,6 @@ class FacebookPixel {
     });
   }
 
-  identify(rudderElement) {
```

Don't simply remove the API as there could be someone using it. Just simply log an error message and return from there.

---

We validated that `identify` hasn't been working post `fbq` init right. So this function is anyway not working. 
To be extra cautious we can still keep this and mark as depreciating with warning from `our` logger as well as mention this clearly in docs

---

Oh. Earlier, @shrouti1507 mentioned that the updated user traits were being sent in the further events. Otherwise, the plan sounds good.

---

@shrouti1507 can you please confirm

---

it will send the updated traits only if the page is reloaded.

---

If there is page load it will always first call the `init` in our sdk then move to specific APIs hence this dosen't have any effect

---

yes. Then can we move forward with [this](https://github.com/rudderlabs/rudder-sdk-js/pull/788#discussion_r1056314423)

---

For now lets throw an error from here that FB PIXEL identify API is deprecated